### PR TITLE
[3][com_tags] Created Date cannot change on every save

### PR DESCRIPTION
**Pull request by zero-24**
Pull Request for Issue #35724  .

### Summary of Changes

Backports https://github.com/joomla/joomla-cms/pull/35724 into 3.10 thanks @alikon 

### Testing Instructions

1.   Set your server time (System - Global Configuration - Server) to something different from UTC. Testing this is more intuitive, if your server is actually located in this time zone (e.g. CEST = UTC+2). Otherwise, you'd have to do the appropriate calculations in order to check if the time fits.
2.   Check the time zone setting of your own user (top right: User Menu - Edit Account - Basic Settings) to match your actual time zone (or to compensate for the "wrong" server time zone from step 1 if your server is on UTC).
3.   At Components - Tags, create a new tag and save it.
4.   Edit the tag and look at the created and modified date.
5.   Save the tag multiple times.

### Actual result BEFORE applying this Pull Request
The Created Date is increased on every save.

### Expected result AFTER applying this Pull Request
The Created Date  do not change on every save.

### Documentation Changes Required
No

### CODE REVIEW

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on a5c7081ffa9d396b5f42b579cce1b801d7d6c09e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35846">issues.joomla.org/tracker/joomla-cms/35846</a>.</sub>

**Comment by alikon** [comment]
I have tested this item :white_check_mark: successfully on a5c7081ffa9d396b5f42b579cce1b801d7d6c09e<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35846">issues.joomla.org/tracker/joomla-cms/35846</a>.</sub>

**Comment by alikon** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/35846">issues.joomla.org/tracker/joomla-cms/35846</a>.</sub>

File: administrator/components/com_tags/models/tag.php [hunk 1]
```diff
@@ -133,20 +133,9 @@ public function getItem($pk = null)
 			$registry = new Registry($result->urls);
 			$result->urls = $registry->toArray();
 
-			// Convert the created and modified dates to local user time for display in the form.
+			// Convert the modified dates to local user time for display in the form.
```
Comment (PhilETaylor) [inline administrator/components/com_tags/models/tag.php]: Singular not plural.

```suggestion
			// Convert the modified date to local user time for display in the form.
```

**Comment by richard67** [comment]
Thanks.

**Pull request merged**